Thumbprint/Targets/Thumbprint/Components/RadioStack.swift

```diff
@@ -0,0 +1,69 @@
+import Combine
+import UIKit
+
+// MARK: - Radio Stack
+/// A convenience class for displaying a group of text only radioViews with consistent styling and spacing.
+public final class RadioStack: UIView, UIContentSizeCategoryAdjusting {
+    /// The radios belonging to this stack
+    private let radioViews: [Radio]
+
+    public var spacing: CGFloat {
+        get { stack.spacing }
+        set { stack.spacing = newValue }
+    }
+
+    /// Sets the numberOfLines of each `Radio` in the stack.
+    public var numberOfLines: Int = 1 {
+        didSet {
+            radioViews.forEach { $0.numberOfLines = numberOfLines }
+        }
+    }
+
+    public var adjustsFontForContentSizeCategory: Bool {
+        didSet {
+            guard adjustsFontForContentSizeCategory != oldValue else { return }
+
+            for radio in radioViews {
+                radio.adjustsFontForContentSizeCategory = adjustsFontForContentSizeCategory
+            }
+        }
+    }
+
+    private let stack: UIStackView
+
+    /// A radio group that manages the radios and vends the selected index.
+    public let radioGroup: RadioGroup<Int>
+
+    /// Initializes a new RadioStack with the given radio titles
+    ///
+    /// - Parameters:
+    ///   - titles: An array of titles with which to create the radio buttons
+    ///   - adjustsFontForContentSizeCategory: Boolean indicating whether the radios in this stack should support Dynamic Type.
+    public init(titles: [String], adjustsFontForContentSizeCategory: Bool = true) {
+        self.radioGroup = RadioGroup()
+        self.radioViews = titles.map({ Radio(text: $0, adjustsFontForContentSizeCategory: adjustsFontForContentSizeCategory) })
+        self.adjustsFontForContentSizeCategory = adjustsFontForContentSizeCategory
+
+        self.stack = UIStackView(arrangedSubviews: radioViews)
+        stack.axis = .vertical
+        stack.alignment = .leading
+        stack.spacing = Space.three
+
+        super.init(frame: .null)
+
+        radioViews.enumerated().forEach { index, radio in
+            radioGroup.registerRadio(radio, forKey: index)
```

Comment:
Might be nice to add an additional constructor that takes a list of `(title, key)` tuples so that people can use `RadioStack` with more specific keys than just indices (and then turn this into a convenience initializer that calls into that

Reply:
We can do it the opposite way, adding a convenience init in TT that does that and if it proves commonly used graduate it to Thumbprint. But on its face a good idea.

Reply:
But my point is that `RadioStack` doesn't currently expose a public setter for the IDs post-initialization, so I don't think it's possible to define such a convenience init in TT right now

Reply:
Uhm it could be a good idea to upgrade `RadioStack` so it doesn't necessarily work on ints. On the other hand I'm kind of not thinking much about it as it doesn't allow for UI configurability and is thus not terribly useful as is right now. We'd need to remake the class to work better for our needs.

Reply:
Maybe we can just add an initializer that takes an already configured `RadioGroup`. Still would need to make `RadioStack` into a generic type but its change would be minimal.